server/data/data-jdbc/src/main/java/org/apache/james/mailrepository/jdbc/JDBCMailRepository.java

```diff
@@ -721,6 +721,27 @@ protected void internalRemove(MailKey key) throws MessagingException {
         }
     }
 
+    @Override
+    public long size() throws MessagingException {
+        Connection conn = null;
```

Comment:
AFAIK all jdbc objects can by handled with a try-with-resource

Reply:
Sure! I was trying to follow common pattern used in the class.

Reply:
In these cases, we often do a first commit to migrate code to the right code-style, then a second one with the change we want.